sdk/python/docs/V1beta1ExperimentStatus.md

```diff
@@ -1,11 +1,11 @@
-# V1alpha3ExperimentStatus
+# V1beta1ExperimentStatus
 
 ## Properties
 Name | Type | Description | Notes
 ------------ | ------------- | ------------- | -------------
 **completion_time** | [**V1Time**](V1Time.md) | Represents time when the Experiment was completed. It is not guaranteed to be set in happens-before order across separate operations. It is represented in RFC3339 form and is in UTC. | [optional] 
```

Comment:
Seems V1Time.md missed? or is not generated. I guess that should be have traceback if this is not generated.

Reply:
@jinchihe Yes,  `V1Time` is not generated. Because it belongs to kubernetes/apimachinery rather than katib.  `V1UnstructuredUnstructured` has the same problem. I can move their definitions from md files, but maybe the better way is adding them manually, just like tf-operator ?

Reply:
I've applied the second approach.

Reply:
I remember we need add related flag and then generate that. But I think that's OK to add that manually, that should be work.